cmake/org.eclipse.cdt.cmake.core/src/org/eclipse/cdt/cmake/core/internal/IOsOverridesSelector.java

```diff
@@ -19,8 +19,9 @@
  * project build.
  *
  * @author Martin Weber
+ * @noimplement
```

Comment:
Unneeded on internal interfaces
```suggestion
```

Was the intention to make this interface API? If so, it needs to be moved to a non-internal namespace.

Reply:
No!
Without this change I get:
"The type org.eclipse.cdt.cmake.core.internal.IOsOverridesSelector is not visible"
It was merely to facilitate moving CMakeBuildConfiguration to API.

Reply:
> "The type org.eclipse.cdt.cmake.core.internal.IOsOverridesSelector is not visible"

I can only see this error if I don't include the `public` on the line below. This review comment was only about the `@noimplement` which shouldn't be applied to internal classes/interfaces as they are implicitly `@noimplement` already